lxd/vm_qemu.go

```diff
@@ -557,13 +588,15 @@ func (vm *vmQemu) Start(stateful bool) error {
 	}
 
 	args := []string{
+		"-S",
```

Comment:
@stgraber this delays the CPU start until a specific command is sent to the qmp, out of interest why is this needed? Thanks

Reply:
So there is no possible race that would cause us to miss some events.